app.py

```diff
@@ -1,4 +1,4 @@
-import streamlit as st
++import streamlit as st
```

Comment:
I guess this plus sign should e removed :)

```suggestion
import streamlit as st
```